webrtc.html

```diff
@@ -1558,6 +1571,18 @@
             the <a href="#dom-peerconnection-ice-gathering-state"><code>RTCPeerConnection</code>
             ice gathering state</a> changes.
           </dd>
+
+          <dt>attribute EventHandler onconnectionstatechange</dt>
+
+          <dd>
+            This event handler, of event handler event type <code><a href=
+            "#event-connectionstatechange">connectionstatechange</a></code>,
+            MUST be fired by all objects implementing the
+            <code><a>RTCPeerConnection</a></code> interface. It is
+            called any time the <a href=
+            "#dom-peerconnection-connection-state"><code>RTCPeerConnection</code>.connectionState</a>
```

Comment:
I will do a PR that removes all the "it is called when" text from EventHandler attribute descriptions. But that shouldn't stop this PR from landing.